classes/services/PKPSubmissionService.php

```diff
@@ -0,0 +1,646 @@
+<?php
+
+/**
+ * @file classes/services/PKPSubmissionService.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionService
+ * @ingroup services
+ *
+ * @brief Helper class that encapsulates submission business logic
+ */
+
+namespace App\Services;
+
+use \DBResultRange;
+use \Application;
+use \DAOResultFactory;
+use \DAORegistry;
+
+import('lib.pkp.classes.db.DBResultRange');
+
+abstract class PKPSubmissionService {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {}
```

Comment:
`\OJS\Services\SubmissionService` extends `\PKP\Services\PKPSubmissionService` and calls `parent::__construct()`. Even if this constructor does nothing at the moment, I suggest we leave it so that parent class keeps the ability to initialize object in the future.